### PR TITLE
(DOCSP-33533) Follow-up small edits

### PR DESCRIPTION
**Pull request by JuliaMongo**
# Pull Request Process

Please read the
[Style Guide Review Process](https://wiki.corp.mongodb.com/display/DE/Technical+Writer+Style+Guide+Review+Process)
wiki page.

Contributors should take the following actions:

- Request a PR review in #docs-style-guide Slack channel.
- Add the Style Guide team reps as reviewers.
- If any reviewer requests changes, address them and request another review.
- When you receive PR review approvals from all the Style Guide team reps,
  they will merge the PR and notify you of the merge in the Slack channel.

# Pull Request Description
This is a small PR that does the following:
Adds structure to a few topics to show sections in the on-page TOC
Fixes one typo in one topic's intro
Replaces apostrophes with curly ones
Edits some text for conciseness without changing meaning.  

JIRA URL: <https://jira.mongodb.org/browse/DOCSP-33533>

Staging URL: <https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-xlarge/NNNNN/>

### CODE REVIEW

File: source/style-guide/style/links-and-cross-references.txt [hunk 1]
```diff
@@ -75,38 +75,35 @@ about MongoDB concepts can click through and explore the linked content.
...
-You may link to the same target multiple times on the same page, but
-think carefully about whether it's absolutely necessary. For example, if
-a page is particularly long and contains many sections, then it might be
-helpful to link to a target an additional time or two. Users may
-immediately navigate to a section far down on a page and miss the
-initial reference.
+  If a page is long and contains many sections, you may link to a target
+  once or twice from this page. This helps users who navigate to a section
+  far down on a page and may miss the initial reference at the top of the
+  page. However, don't link to a target more than twice on the same page.
```
Comment (corryroot) [inline source/style-guide/style/links-and-cross-references.txt:101]: I think this is one example scenario, but might not be the only scenario where this would be appropriate.

```suggestion
  You may link to the same target multiple times on the same page, but
  think carefully about whether it's absolutely necessary. For example, if
  a page is particularly long and contains many sections, then it might be
  helpful to link to a target an additional time or two. Users may
  immediately navigate to a section far down on a page and miss the
  initial reference.
```

**Review by corryroot — CHANGES_REQUESTED** [review]
Thanks @JuliaMongo! I commented with my thoughts.

File: source/style-guide/style/callouts.txt [hunk 2]
```diff
@@ -78,18 +79,25 @@ When creating callouts, use the following guidelines:
    attempt to create callouts using boldface, headings, or custom
    :abbr:`CSS (Cascading Style Sheets)`.
 
+- Don't stack multiple admonitions directly after one another, as these
+  can be distracting for users. To avoid stacking admonitions, group
```
Comment (corryroot) [inline source/style-guide/style/callouts.txt:83]:
```suggestion
  can distracti users. To avoid stacking callouts, group
```

File: source/style-guide/style/callouts.txt [hunk 3]
```diff
@@ -78,18 +79,25 @@ When creating callouts, use the following guidelines:
    attempt to create callouts using boldface, headings, or custom
    :abbr:`CSS (Cascading Style Sheets)`.
 
+- Don't stack multiple admonitions directly after one another, as these
```
Comment (corryroot) [inline source/style-guide/style/callouts.txt:82]: Let's stick with "callouts" for now.

```suggestion
- Don't stack multiple callouts directly after one another, as these
```

File: source/style-guide/style/callouts.txt [hunk 4]
```diff
@@ -78,18 +79,25 @@ When creating callouts, use the following guidelines:
...
+  can be distracting for users. To avoid stacking admonitions, group
+  supplemental information from admonitions into a dedicated section.
```
Comment (corryroot) [inline source/style-guide/style/callouts.txt:84]:
```suggestion
  supplemental information from callouts into a dedicated section.
```

File: source/style-guide/style/callouts.txt [hunk 5]
```diff
@@ -78,18 +79,25 @@ When creating callouts, use the following guidelines:
...
+
+- Don't use admonitions in tables and option lists, if possible, because
```
Comment (corryroot) [inline source/style-guide/style/callouts.txt:86]:
```suggestion
- Don't use callouts in tables and option lists, if possible, because
```

File: source/style-guide/style/callouts.txt [hunk 6]
```diff
@@ -78,18 +79,25 @@ When creating callouts, use the following guidelines:
...
+  admonitions reduce scannability of tables and options lists.
```
Comment (corryroot) [inline source/style-guide/style/callouts.txt:87]:
```suggestion
  callouts reduce the scannability of tables and options lists.
```

File: source/style-guide/style/callouts.txt [hunk 7]
```diff
@@ -78,18 +79,25 @@ When creating callouts, use the following guidelines:
...
 
 -  Avoid nesting code blocks inside callouts.
 
 -  Place a callout as close as possible to the information that it
    emphasizes or clarifies.
 
--  Don't "stack" callouts of the same type (for example, by following
-   one labeled note directly with another labeled note). Instead, use
+-  Avoid "stack" callouts of the same type. For example, avoid adding
+   one labeled note directly after another labeled note. Instead, use
```
Comment (corryroot) [inline source/style-guide/style/callouts.txt:97]:
```suggestion
-  Avoid stacking callouts of the same type. For example, avoid adding
   one labeled note directly after another labeled note. Instead, use
```

File: source/style-guide/style/callouts.txt [hunk 8]
```diff
@@ -78,18 +79,25 @@ When creating callouts, use the following guidelines:
...
    separate paragraphs or an unordered list within a single callout.
    You can have callouts of different types follow one
-   another.
```
Comment (corryroot) [inline source/style-guide/style/callouts.txt:92]: I recommend we remove this. Too contradictory to above.

File: source/style-guide/style/links-and-cross-references.txt [hunk 9]
```diff
@@ -75,38 +75,30 @@ about MongoDB concepts can click through and explore the linked content.
...
-into a dedicated section.
+  If users follow inline references while in the middle of a procedure,
+  they might have difficulty navigating back and reorienting themselves
+  with their current step.
 
-Inline textual references may be appropriate for more reference-heavy
-pages. Admonitions can potentially reduce scannability of tables and
-options lists, and should therefore be avoided in those contexts
-whenever possible.
+  Inline textual references may be appropriate for more reference-heavy pages.
```
Comment (corryroot) [inline source/style-guide/style/links-and-cross-references.txt:84]: Remove bullet, now that we have only one item.

File: source/style-guide/style/links-and-cross-references.txt [hunk 10]
```diff
@@ -75,38 +75,30 @@ about MongoDB concepts can click through and explore the linked content.
...
-a page is particularly long and contains many sections, then it might be
-helpful to link to a target an additional time or two. Users may
-immediately navigate to a section far down on a page and miss the
-initial reference.
+  You may link to the same target more than once on the same page, but
+  think carefully about whether it's absolutely necessary. For example,
+  if a page is long and contains many sections, then it might be helpful
+  to link to a target once or twice from this page to help users who
+  navigate to a section far down on a page and may miss the initial
+  reference at the top of the page.
```
Comment (corryroot) [inline source/style-guide/style/links-and-cross-references.txt:96]: I think I still prefer the original wording for this bit.

**Review by corryroot — COMMENTED** [review]
Thanks @JuliaMongo! I commented with a few more suggestions.

File: source/style-guide/style/links-and-cross-references.txt (hunk 10 above)
Comment (JuliaMongo) [reply source/style-guide/style/links-and-cross-references.txt:96]: I reverted it. I think it is a bit too wordy, but I will go with your preference!

File: source/style-guide/style/links-and-cross-references.txt [hunk 11]
```diff
@@ -75,38 +69,30 @@ about MongoDB concepts can click through and explore the linked content.
...
-into a dedicated section.
+If users follow inline references while in the middle of a procedure,
+they might have difficulty navigating back and reorienting themselves
+with their current step.
 
-Inline textual references may be appropriate for more reference-heavy
-pages. Admonitions can potentially reduce scannability of tables and
-options lists, and should therefore be avoided in those contexts
-whenever possible.
+Inline textual references may be appropriate for more reference-heavy pages.
```
Comment (corryroot) [inline source/style-guide/style/links-and-cross-references.txt:78]: I think a single paragraph is appropriate here.

File: source/style-guide/style/links-and-cross-references.txt [hunk 12]
```diff
@@ -75,38 +69,30 @@ about MongoDB concepts can click through and explore the linked content.
...
-You may link to the same target multiple times on the same page, but
-think carefully about whether it's absolutely necessary. For example, if
-a page is particularly long and contains many sections, then it might be
-helpful to link to a target an additional time or two. Users may
-immediately navigate to a section far down on a page and miss the
-initial reference.
+  You may link to the same target multiple times on the same page, but
+  think carefully about whether it's absolutely necessary. For example,
+  if a page is particularly long and contains many sections, then it might
+  be helpful to link to a target an additional time or two. Users may
```
Comment (corryroot) [inline source/style-guide/style/links-and-cross-references.txt:88]:
```suggestion
  help to link to a target an additional time or two. Users might
```

File: source/style-guide/writing/index.txt [hunk 13]
```diff
@@ -23,6 +23,7 @@ and consistent, and that helps users accomplish their goals.
    /style-guide/writing/use-gender-neutral-pronouns
    /style-guide/writing/use-positive-statements
    /style-guide/writing/use-correct-punctuation
+   /style-guide/style/voice-and-tone
```
Comment (corryroot) [inline source/style-guide/writing/index.txt:26]: Move back to Style.

**Review by corryroot — CHANGES_REQUESTED** [review]
Thanks @JuliaMongo. Please move Voice and Tone back to the Style section. There's a small distinction between the pages in the Writing section and the pages in the Style section. Style is more of a reference.

File: source/style-guide/writing/index.txt (hunk 13 above)
Comment (JuliaMongo) [reply source/style-guide/writing/index.txt]: I suggested moving it because it is not easy to find as of now. Navigation in our style guide needs more work, I think you would agree.

**Comment by JuliaMongo** [comment]
Back to you, @corryroot and thank you for the reviews!

File: source/style-guide/style/glossaries.txt [hunk 14]
```diff
@@ -14,12 +20,6 @@ Create a glossary to document the following items:
 
 This section provides guidelines for the following items:
```
Comment (corryroot) [inline source/style-guide/style/glossaries.txt:21]: This is a dangling stem sentence now.

File: source/style-guide/style/links-and-cross-references.txt [hunk 15]
```diff
@@ -4,12 +4,6 @@
 Links and Cross-References
 ==========================
 
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 2
-   :class: singlecol
```
Comment (corryroot) [inline source/style-guide/style/links-and-cross-references.txt:11]: Why did you delete this?

File: source/style-guide/style/links-and-cross-references.txt (hunk 11 above)
Comment (corryroot) [reply source/style-guide/style/links-and-cross-references.txt:78]: Unresolved.

File: source/style-guide/style/links-and-cross-references.txt (hunk 12 above)
Comment (corryroot) [reply source/style-guide/style/links-and-cross-references.txt:88]: Unresolved.

File: source/style-guide/writing/index.txt (hunk 13 above)
Comment (corryroot) [reply source/style-guide/writing/index.txt]: It's more of a reference pages and links to the related pages in the writing section. It still belongs here.

File: source/style-guide/style/links-and-cross-references.txt [hunk 16]
```diff
@@ -75,38 +69,30 @@ about MongoDB concepts can click through and explore the linked content.
...
-their current step.
+Avoid adding inline textual references in tutorials or stepped procedures. 
 
-Additionally, avoid stacking multiple admonitions directly after one
-another, as these can be distracting for users. To avoid stacking
-admonitions, consider grouping supplemental information from admonitions
-into a dedicated section.
+If users follow inline references while in the middle of a procedure,
+they might have difficulty navigating back and reorienting themselves
+with their current step.
```
Comment (lindseymoore) [inline source/style-guide/style/links-and-cross-references.txt:76]:
```suggestion
If users follow inline references while in the middle of a procedure,
they might have difficulty navigating back to the procedure and reorienting themselves.
```
Suggestion for clarity

File: source/style-guide/style/tables.txt [hunk 17]
```diff
@@ -6,17 +6,17 @@ Tables
...
-guidelines for the following aspects of tables:
-
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
+Text that's difficult to read in paragraph form often becomes clear
```
Comment (lindseymoore) [inline source/style-guide/style/tables.txt:15]:
```suggestion
Text that's difficult to read in paragraph form often becomes more clear
```
Nit

**Review by lindseymoore — APPROVED** [review]
Small things, LGTM! Thanks for making these changes!

File: source/style-guide/style/callouts.txt [hunk 18]
```diff
@@ -78,18 +78,23 @@ When creating callouts, use the following guidelines:
    attempt to create callouts using boldface, headings, or custom
    :abbr:`CSS (Cascading Style Sheets)`.
 
+- Don't stack multiple callouts directly after one another, as these can
+  distract users. To avoid stacking callouts, group supplemental information
+  from callouts into a dedicated section.
+
+- Don't use callouts in tables and option lists, if possible, because
+  callouts reduce the scannability of tables and options lists.
```
Comment (jeff-allen-mongo) [inline source/style-guide/style/callouts.txt:86]:
```suggestion
- If possible, don't use callouts in tables and option lists.
  Callouts reduce the scannability of tables and options lists.
```

File: source/style-guide/style/callouts.txt [hunk 19]
```diff
@@ -78,18 +78,23 @@ When creating callouts, use the following guidelines:
...
 -  Avoid nesting code blocks inside callouts.
 
 -  Place a callout as close as possible to the information that it
    emphasizes or clarifies.
 
--  Don't "stack" callouts of the same type (for example, by following
-   one labeled note directly with another labeled note). Instead, use
+-  Avoid stacking callouts of the same type. For example, avoid adding
+   one labeled note directly after another labeled note. Instead, use
    separate paragraphs or an unordered list within a single callout.
```
Comment (jeff-allen-mongo) [inline source/style-guide/style/callouts.txt:97]:
```suggestion
-  Avoid stacking callouts of the same type. For example, avoid adding
   one labeled note directly after another labeled note. Instead, use
   a single callout that contains separate paragraphs or an unordered list.
```

File: source/style-guide/style/tables.txt (hunk 17 above)
Comment (jeff-allen-mongo) [reply source/style-guide/style/tables.txt:15]: Perhaps "clearer"?

File: source/style-guide/style/callouts.txt [hunk 20]
```diff
@@ -6,9 +6,8 @@ Callouts and Admonitions
 
 .. default-domain:: mongodb
 
-Callouts or admonitions emphasize important or
-helpful information. Use callouts sparingly to avoid clutter on
-the page and to preserve visual impact. 
+Callouts or admonitions emphasize important or helpful information. Use
+callouts sparingly to avoid clutter on the page and to preserve visual impact.
```
Comment (jeff-allen-mongo) [inline source/style-guide/style/callouts.txt:10]: Consider rephrasing the second sentence in the form to (x), (y):

```suggestion
Callouts or admonitions emphasize important or helpful information. To
avoid clutter and to preserve visual impact, use callouts sparingly. 
```

**Review by jeff-allen-mongo — APPROVED** [review]
Leaving a few nits but overall this LGTM. Thanks @JuliaMongo

File: source/style-guide/style/links-and-cross-references.txt (hunk 12 above)
Comment (JuliaMongo) [reply source/style-guide/style/links-and-cross-references.txt:88]: Addressed.

File: source/style-guide/style/glossaries.txt (hunk 14 above)
Comment (JuliaMongo) [reply source/style-guide/style/glossaries.txt:21]: Fixed.

File: source/style-guide/style/links-and-cross-references.txt (hunk 15 above)
Comment (JuliaMongo) [reply source/style-guide/style/links-and-cross-references.txt:11]: Deleted by mistake, restored.

File: source/style-guide/style/callouts.txt (hunk 20 above)
Comment (corryroot) [reply source/style-guide/style/callouts.txt:10]: @JuliaMongo, did you want to incorporate this feedback?

**Review by corryroot — APPROVED** [review]
LGTM!

File: source/style-guide/style/callouts.txt (hunk 20 above)
Comment (JuliaMongo) [reply source/style-guide/style/callouts.txt]: Yes, I just forgot to save. Addressed.
